network/network_linux_test.go

```diff
@@ -0,0 +1,98 @@
+package network
+
+import (
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/Azure/azure-container-networking/platform"
+)
+
+func TestAddDNSServersWithErr(t *testing.T) {
```

Comment:
This checks for empty dnsServers length case right ? Ifo so, the result of the error is `nil` so not sure if `TestAddDNSServersWithErr` make sense as we don't expect an error ?